### PR TITLE
Bump CUDA version in cuda11x-cuda-python CI

### PR DESCRIPTION
**Pull request by asi1024**
Fix https://ci.preferred.jp/cupy.linux.cuda11x-cuda-python/178207/.

### CODE REVIEW

**Comment by asi1024** [comment]
https://ci.preferred.jp/cupy.linux.cuda11x-cuda-python/178256/

**Comment by asi1024** [comment]
https://ci.preferred.jp/cupy.linux.cuda11x-cuda-python/178258/

**Review by kmaehashi — APPROVED** [review]
Thanks, LGTM! Please proceed to merge and backport after test pass.

File: cupyx/_runtime.py [hunk 1]
```diff
@@ -111,8 +111,9 @@ def __init__(self, *, full=True):
         # CUDA Driver
         self.cuda_build_version = str(cupy.cuda.driver.get_build_version())
         if cupy.cuda.driver._is_cuda_python():
-            import cuda
-            self.cuda_build_version += f' (CUDA Python: {cuda.__version__})'
+            import cuda.bindings
```
Comment (leofang) [inline cupyx/_runtime.py:114]: We will need a try-except here. This only works if users are using latest cuda-python 11.8.5 or 12.6.2. Older versions don't have this new module yet.

**Review by leofang — CHANGES_REQUESTED** [review]

**Review by leofang — APPROVED** [review]
Thanks! LGTM!

**Pull request merged**